dom.bs

```diff
@@ -5701,11 +5719,13 @@ invoked, must return a new {{DocumentFragment}} <a>node</a> whose <a for=Node>no
 [Exposed=Window]
 interface ShadowRoot : DocumentFragment {
   readonly attribute ShadowRootMode mode;
+  readonly attribute SlotAssignmentMode slotAssignment;
   readonly attribute Element host;
   attribute EventHandler onslotchange;
 };
 
 enum ShadowRootMode { "open", "closed" };
+enum SlotAssignmentMode { "manual", "auto" };
```

Comment:
Perhaps we want to future-proof this and call "auto" as "name" instead.
That way, we can add more *automatic" mode like based on tag name, etc...

Reply:
This is a good point. However, I'm not sure "name" is meaningful enough. 
How about "default" -- indicating default slotting?

Reply:
Update spec to use "name"